workflow-templates/im-deploy-windows-files.yml

```diff
@@ -0,0 +1,308 @@
+# Workflow Code: MysteriousStranger_v1    DO NOT REMOVE
+# Purpose:
+#    Gathers various stakeholder and attestor approvals, downloads artifacts from a release with the
+#    specified tags, makes changes to any configuration files for the specified environments, stops
+#    the service, deploys the pre-built artifacts to a target machine then starts the service. Each
+#    deployable project in a solution should have their own deployment workflow.
+#
+# Frequency:
+#    Each set of deployable files in the repository should have their own copy of this workflow rather than trying to deploy several
+#    sets of files with the same workflow.  If the service is deployed as part of a group, the service still needs its own copy of this
+#    workflow but it will be streamlined (instructions below) since the Deploy Multiple Items workflow handles many of these checks.
+#
+# TODO: Prerequisites:
+#     - Make sure the Service account secrets have been added to the environment
+#     - Make sure the host windows server has been prepped to accept incoming WinRM Requests
```

Comment:
Is this point the same as the last point? Both talk about WinRM being necessary, but the last point explains how to set it up. Could we get away with only the last point?

Reply:
You're right. It's best to remove this point and keep the last one because it's more explicit.